Project.toml

```diff
@@ -0,0 +1,10 @@
+name = "Roots"
+uuid = "f2b01f46-fcfa-551c-844a-d8ac1e96c665"
+
+[deps]
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Statistics = "1c261c3c-9a38-5c8c-b229-00278e4435fb"
```

Comment:
Where did this UUID come from? The one for the Statistics stdlib is `10745b16-79ce-11e8-11f9-7d13ad32a3b2`.

Reply:
Interesting. I used Stefan's new script to generate this. Should I hand edit or rerun?

Reply:
Might as well just hand-edit. Though I write all of my Project.toml files by hand, so do whatever you're more comfortable with.

Reply:
Admittedly, I was looking for the easiest way out. It seems like it may be hand editing... Thanks for the feedback!